### PR TITLE
add list streams

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,18 @@ $stream->name; // string with the name
...
+You can get information about all the streams in a session by calling the `listStreams($sessionId)` method of the
+`OpenTok\OpenTok` class.
+
+```php
+use OpenTok\Session;
+
+// Get list of streams from just a sessionId (fetched from a database)
+$streamList = $opentok->listStreams($sessionId);
+
+$streamList->totalCount(); // total count
```
Comment (JayTokBox) [inline README.md]: The readme for other SDK does not document all exposed methods

File: src/OpenTok/Session.php [hunk 2]
```diff
@@ -144,7 +144,7 @@ public function generateToken($options = array())
     {
         return $this->opentok->generateToken($this->sessionId, $options);
     }
-    
+
```
Comment (JayTokBox) [inline src/OpenTok/Session.php]: extra extra !!!

File: README.md [hunk 3]
```diff
@@ -130,6 +130,18 @@ $stream->name; // string with the name
...
+```
+
```
Comment (JayTokBox) [inline README.md]: extra line

File: src/OpenTok/OpenTok.php [hunk 4]
```diff
@@ -520,6 +521,25 @@ public function getStream($sessionId, $streamId)
...
+     * 
+     * @param String $sessionId The session ID.
+     *
+     * @return StreamList A StreamList object. Call the items() method of the StreamList object
+     * to return an array of Stream objects.     
+     */
+
+    public function listStreams($sessionId)
+    {
+        Validators::validateSessionIdBelongsToKey($sessionId, $this->apiKey);
```
Comment (JayTokBox) [inline src/OpenTok/OpenTok.php]: why validate with api key?

File: src/OpenTok/StreamList.php [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+namespace OpenTok;
+
+class StreamList {
+
+    private $data;
+
+    public function __construct($streamListData)
+    {
+
```
Comment (JayTokBox) [inline src/OpenTok/StreamList.php]: extra line

File: src/OpenTok/OpenTok.php (hunk 4 above)
Comment (msach22) [reply src/OpenTok/OpenTok.php]: Validating for because the jwt must be signed with the right `apiKey` and `secret`

File: README.md (hunk 1 above)
Comment (msach22) [reply README.md]: Good point, will add them for other methods in a separate PR

File: README.md (hunk 3 above)
Comment (msach22) [reply README.md]: There for readability 😄

File: src/OpenTok/OpenTok.php (hunk 4 above)
Comment (JayTokBox) [reply src/OpenTok/OpenTok.php]: we don't do this everywhere and is not consistent with REST codes.
approving with that reservation

**Pull request merged**